shorten-url.ts

```diff
@@ -0,0 +1,18 @@
+
+export async function shortenURLs(longURL: string): Promise<string> {
+  
+  const response = await fetch("https://gotiny.cc/api", {
```

Comment:
This one should also be added to the .env file. Usually when integrating with external systems, the urls are different per environment.